manageiq-providers-autosde.iml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="RUBY_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="jdk" jdkName="ruby-2.6.8-p205" jdkType="RUBY_SDK" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="PROVIDED" name="bundler (v1.17.2, ruby-2.6.8-p205) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="ethon (v0.15.0, ruby-2.6.8-p205) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="ffi (v1.15.5, ruby-2.6.8-p205) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="typhoeus (v1.4.0, ruby-2.6.8-p205) [gem]" level="application" />
+  </component>
+  <component name="RakeTasksCache">
+    <option name="myRootTask">
+      <RakeTaskImpl id="rake" />
+    </option>
+  </component>
+</module>
```

Comment:
Please add `*.iml` to your preferably your global `.gitignore` file so they aren't added automatically

Reply:
will do. sorry about that.

Reply:
You still need to delete this file

Reply:
done

Reply:
Still still need to delete this file :wink:

Reply:
In each rebase this file come back to life 👻
I will delete it again

Reply:
Recommend rebasing the original commit bb7058bd72e4845b5eadfaf22a295d7b83b7dae1 and removing the file from there so we aren't tracking an add+remove of a file we don't want